packages/@office-iss/react-native-win32/src/Libraries/Components/View/ViewAccessibility.win32.d.ts

```diff
@@ -76,10 +76,10 @@ export interface AccessibilityProps
    */
   accessibilityValue?: AccessibilityValue | undefined;
 
-  'aria-valuemax'?: AccessibilityValue['max'];
-  'aria-valuemin'?: AccessibilityValue['min'];
-  'aria-valuenow'?: AccessibilityValue['now'];
-  'aria-valuetext'?: AccessibilityValue['text'];
+  'aria-valuemax'?: AccessibilityValue['max'] | undefined;
```

Comment:
Are you adding these `| undefined` marks manually? Is it worth commenting these as `//TODO Windows Fabric`?

Reply:
Why would we say todo fabric?  These work today in paper.

Reply:
And the `| undefined` is required by the newer more strict flow config.

Reply:
These weren't added manually, taken from upstream!